test/__tests__/TypeaheadAnswer.test.js

```diff
@@ -1,89 +0,0 @@
-import React from 'react';
```

Comment:
@LaChope, what is the reason for removing this test?

Reply:
@blcham This test was testing only external library where we can't do anything, I then thought it was not useful to keep it, what do you think?

Reply:
Not sure what you mean. It tests TypeaheadAnswer which is part of the project:
```
import TypeaheadAnswer from '../../src/components/answer/TypeaheadAnswer';
```